rest/taskrouter/workflows/instance/get/example-1/example-1.5.x.php

```diff
@@ -17,4 +17,4 @@
     ->workflows($workflowSid)
     ->fetch();
 
-echo $workflow->friendlyName;
+echo $workflow->friendly_name;
```

Comment:
snake_case is used in v4 of the twilio library, but camelCase was correct in v5.x snippets